src/aind_data_transfer_service/configs/job_upload_template.py

```diff
@@ -57,42 +80,57 @@ class JobUploadTemplate:
             "/allen/aind/stage/fake/dir",
         ],
     ]
-    VALIDATORS = [
-        {
-            "name": "platform",
-            "type": "list",
-            "options": [p().abbreviation for p in Platform._ALL],
-            "column_indexes": [HEADERS.index("platform")],
-        },
-        {
-            "name": "modality",
-            "type": "list",
-            "options": [m().abbreviation for m in Modality._ALL],
-            "column_indexes": [
-                HEADERS.index("modality0"),
-                HEADERS.index("modality1"),
-            ],
-        },
-        {
-            "name": "datetime",
-            "type": "date",
-            "column_indexes": [HEADERS.index("acq_datetime")],
-        },
-    ]
 
-    @staticmethod
-    def create_job_template():
+    @property
+    def validators(self) -> List[Dict[str, Any]]:
+        """
+        Returns
+        -------
+        List[Dict[str, Any]]
+          A list of validators for fields that require validation.
+
+        """
+        return [
+            {
+                "name": "platform",
+                "type": "list",
+                "options": [p().abbreviation for p in Platform._ALL],
+                "column_indexes": [self.HEADERS.index("platform")],
+            },
+            {
+                "name": "project_name",
+                "type": "list",
+                "options": [p for p in self.project_names],
```

Comment:
It looks like Excel can only store a max of 255 chars for Data validators defined directly as a list. Is there a way we can shorted these? Or, we may need to write the project names into another worksheet in the same workbook and use a formula to pull them.

Reply:
Removed the validator for the time being